### PR TITLE
Add Musescore

### PR DESCRIPTION
**Pull request by jtagcat**

### CODE REVIEW

File: _data/sites.json [hunk 1]
```diff
@@ -4916,6 +4916,17 @@
...
+        "name": "Musescore",
+        "url": "https://musescore.com/user/settings/profile",
+        "difficulty": "easy",
+        "notes": "Click on 'Delete account' at the left of the page.",
+        "domains": [
+            "musescore.com",
+            "musescore.org"
+        ]
+    },
+
```
Comment (tupaschoal) [inline _data/sites.json]:
```suggestion

```
Remove trailing spaces.

File: _data/sites.json [hunk 2]
```diff
@@ -4916,6 +4916,16 @@
...
+    {
+        "name": "Musescore",
+        "url": "https://musescore.com/user/settings/profile",
+        "difficulty": "easy",
+        "notes": "Click on 'Delete account' at the left of the page.",
+        "domains": [
+            "musescore.com",
+            "musescore.org"
+        ]
+    },
```
Comment (Nightfirecat) [inline _data/sites.json]: Now this is missing the newline afterward. Please add a line (with no trailing space) following this block.

```suggestion
    },

```

Comment (Nightfirecat) [reply _data/sites.json]: This might have been because the suggestion in the previous comment wasn't formatted correctly--apologies for that.

**Comment by tupaschoal** [comment]
Thanks for your contribution!

**Pull request merged**